src/loader.js

```diff
@@ -349,6 +349,7 @@ function pitch(request) {
   const outputOptions = {
     filename: childFilename,
     publicPath,
+    asyncChunkLoading: false,
```

Comment:
We should not do it or fix it on webpack side

Reply:
@alexander-akait will https://github.com/webpack/webpack/pull/17365 fix this ?

Reply:
@ersachin3112 I think no